docker/psibase-contributor.Dockerfile

```diff
@@ -85,6 +85,9 @@ RUN mkdir -p ${PSINODE_PATH}    \
     && git clone https://github.com/gofractally/psibase.git . \
     && git submodule update --init --recursive
 
+# Add locally built rust programs to path
+ENV PATH=${PSINODE_PATH}/build/rust/release:$PATH
```

Comment:
Is this the right way, or would it be better to install `cargo-psibase`?

Reply:
Hmm, I've been thinking that it's better to use the locally built one. Just like we don't install `psinode` and `psibase` from the last psidk release either

Reply:
I meant, should `cargo-psibase` be installed to `psidk/bin` (like `psibase`) instead of being used from the build directory.

Reply:
Oh, I see, that's probably a good idea. Although that would mean that cargo-psibase would be distributed with psidk, right? I'd imagine typically the correct way to install cargo-psibase is using `cargo install` directly.